logging/Notes_on_using_TLS.md

```diff
@@ -2,7 +2,21 @@
 
 ## Overview
 
-TLS is required for communication between the Elasticsearch nodes and for accessing Elasticsearch from other components. It is also a best practice to use TLS (by using HTTPS) when accessing Kibana from a browser.
+TLS enablement for SAS Viya logging is divided into two parts:
+
+- **TLS for in-cluster communications**, which is between logging components within
+the cluster (between the Elasticsearch nodes and between Elasticsearch and other logging components). TLS must be enabled on these connections.
+
+- **TLS for communications into the cluster**, which is between a user's 
+browser and Kibana. TLS is optional on these connections, although enabling
+TLS is a best practice.
+
+The information in this document explains only how to configure TLS 
+for in-cluster communications, either automatically (using cert-manager) 
+or manually.
```

Comment:
The automatic (cert-manager)/manual steps aren't really configuring TLS, they're generating/ensuring the required TLS certificates  are in-place to enable TLS to work.  Might be a nit but cert-manager really doesn't configure anything...the deployment scripts handle the actual configuration work, cert-manager is called upon to generate the certificates.